contrib/Helix.appdata.xml

```diff
@@ -6,27 +6,26 @@
   <name>Helix</name>
   <summary>A post-modern text editor</summary>
   <summary xml:lang="ar">مُحَرِّرُ نُصُوصٍ سَابِقٌ لِعَهدِه</summary>
+  <developer id="com.helix_editor">
+    <name>The Helix Community</name>
```

Comment:
Yeah let's match what we do for the crates themselves and put @archseer https://github.com/helix-editor/helix/blob/2cadec0b1182332338a5a1cc3062776f834d8835/Cargo.toml#L46

Reply:
Ok!